scripts/populatedb_prod.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+EMAIL=
```

Comment:
I think this and the `|| echo` should be moved to either the `populateEndpoints.sh` script or the `automatic_endpoint_update.sh` script!